test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -285,4 +285,15 @@ export default class BasePageObject {
   isVisible() {
     return this.form.el.visible;
   }
+
+  async getAriaDescription(el) {
```

Comment:
Might be a worthwhile place to put a comment describing why this method formats the returned value the way that it does (i.e. this is how screenreaders handle multiple elements in the aria-description by concatenating the values with a space.